ofl/cairoplay/METADATA.pb

```diff
@@ -9,15 +9,14 @@ fonts {
   style: "normal"
   weight: 400
   filename: "CairoPlay[slnt,wght].ttf"
-  post_script_name: "CairoPlay-Regular"
-  full_name: "Cairo Play Regular"
+  post_script_name: "CairoPlay-ExtraLight"
+  full_name: "Cairo Play ExtraLight"
```

Comment:
It looks like the font origin has changed. Regular would be better default and also would preserve the previous name ID 6, reminds me that this check really needs to be implemented: https://github.com/googlefonts/fontbakery/issues/3541